modules/monitoring-disabling-the-local-alertmanager.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="disabling_the_local_alertmanager_{context}"]
+= Disabling the local Alertmanager
+
+A local Alertmanager that routes alerts from Prometheus instances is enabled by default in the `openshift-monitoring` project of the {product-title} monitoring stack.
+
+If you do not need the local Alertmanager, you can disable it by configuring the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project:
```

Comment:
same suggestion as above re using "config map" instead of "`ConfigMap` object" - they mean the same thing, it's up to you though.

Reply:
I like this suggestion. Change made.